admin_guide/scheduling/descheduler.adoc

```diff
@@ -252,6 +257,28 @@ strategies:
 
 <1> Set this value to `enabled: true` to use this policy. Set to `false` to disable this policy.
 
+[[admin-guide-descheduler-policies-node]]
+=== Remove Pods Violating Node Affinity
+
+The `RemovePodsViolatingNodeAffinity` strategy ensures that all pods violating node affinity are removed from nodes. This situation could occur if a node no longer satisfies a pod’s affinity rule. If another node is available that satisfies the affinity rule, the pod is evicted.
+
+For example, *podA* is scheduled on *nodeA*, because the node satisfies the `requiredDuringSchedulingIgnoredDuringExecution` node affinity rule at the time of scheduling. If *nodeA* stops satisfying the rule, when the strategy gets executed, if there is another node available that satisfies the node affinity rule, *podA* is evicted from *nodeA* and moved to the other node.
```

Comment:
There are quite a few independent clauses strung together in these paragraphs. It might improve clarity by using active voice where possible. Not sure if my suggested edit still holds the same meaning, but gave it a shot:

```suggestion
The `RemovePodsViolatingNodeAffinity` strategy ensures that all pods violating node affinity are removed from nodes. This situation might occur if a node no longer satisfies a pod’s affinity rule. If another node is available that satisfies the affinity rule, the pod is evicted.

For example, *podA* is scheduled on *nodeA*, because the node satisfies the `requiredDuringSchedulingIgnoredDuringExecution` node affinity rule at the time of scheduling. If *nodeA* stops satisfying the rule, and if there is another node available that satisfies the node affinity rule, the strategy evicts *podA* from *nodeA* and moves it to the other node.
```